### ISSUE TITLE
Error on User Registration

### ISSUE THREAD

**Issue by milezteg** [infinity-next/infinity-next#123]
On a fresh clone from master I get an error when trying to register an account:

`BadMethodCallException in Controller.php line 283: Method [getRegistrationValidator] does not exist.`

Full trace is here:  http://pastebin.com/gcNrTALW

On the Create a Board page the Captcha no longer appears to be solvable.  Maybe these are both related to recent changes to the captcha code.  Can anyone confirm?

**Comment by jaw-sh** [comment]
Confirmed. Going to see if I can get something out as an emergency. A lot of that area got busted up in my last commit because I'm completely redoing how board owners manage permissions.

**Comment by jaw-sh** [comment]
Apologies. I sent something upstream I shouldn't have. I got too comfy with using the Master branch and now that other people are trying to use the software it's resulting in problems with semi-finished commits.

The problem is really hard to get around, and it'd be time consuming for me to split out my fixes. I'm going to have to wrap up current work and then I will begin version controlling it again.

**Comment by milezteg** [comment]
Sounds good.  Will wait patiently for the new branch =)

**Comment by jaw-sh** [comment]
@milezteg Hey, I think I've got it working. Try pulling master and running `php artsan db:migrate` and `php artisan db:seed` and then `php artisan cache:clear` and begin testing. If you run into any errors or problems let me know.

**Comment by jaw-sh** [comment]
Also, I'm about to commit an update to the captcha. Run composer as well. If you continue having errors with that let me know.

**Comment by milezteg** [comment]
Yeap, I got impatient and installed up to here yesterday:  https://github.com/infinity-next/infinity-next/commit/a1ee70350c8171a629a30dbdde90db08dfcdce77 (3 commits back)

It was good well done!  All the main stuff was working.  I'll add the 2 new commits and begin testing.  I noticed something minor but have to remind myself what it was.

As an unrelated side note.  I noticed in the configs the S3 bucket stuff.  Killer feature.  Is that functional?  I want to start testing that as well.

**Comment by jaw-sh** [comment]
I have no idea if it works. That's a default Laravel feature and it should theoretically switch over no problem.

**Comment by milezteg** [comment]
Well I have no idea why but I am back to the captcha being unsolvable.  /me pulls hair.

`The security image was not answered correctly`

I tried rolling back a couple commits but it made no difference.  Is there a dir somewhere that needs write permissions for the captcha?  Hmm I forget.

One thing I noticed is when I did my `composer update` it updated some dependencies including the captcha:

`Updating dependencies (including require-dev)`
`  - Removing laravel/framework (v5.1.18)`
`  - Installing laravel/framework (v5.1.19)`

  `- Removing phpunit/phpunit-mock-objects (2.3.7)`
 ` - Installing phpunit/phpunit-mock-objects (2.3.8)`

 ` - Removing phpunit/phpunit (4.8.9)`
  `- Installing phpunit/phpunit (4.8.10)`

 ` - Removing infinity-next/brennan-captcha (dev-master b223c25)`
`  - Installing infinity-next/brennan-captcha (dev-master 7449e37)`
